config/dol_config/mw_maxl_planner.yaml

```diff
@@ -0,0 +1,81 @@
+mw_maxl_planner:
```

Comment:
change the name of the folder, it should be robot_follower_config ( folder name)